spyder_unittest/tests/test_unittestplugin.py

```diff
@@ -38,6 +38,7 @@ def test_plugin_initialization(plugin):
     assert plugin.main.run_menu_actions[1].text() == 'Run unit tests'
 
 
+@pytest.mark.skip('not clear how to test interactions between plugins')
```

Comment:
> What do you want to check with this test? I can give you a hand to fix it if I understand better what it does.

Broadly speaking, I want to check that if the Python path changes in the Python Path Manager, this change is picked up by the unittest plugin. Can you give me an example of a test that tests the interaction between two plugins? I should be able to take it from there.

Reply:
> What do you want to check with this test? I can give you a hand to fix it if I understand better what it does.

Broadly speaking, I want to check that if the Python path changes in the Python Path Manager, this change is picked up by the unittest plugin. Can you give me an example of a test that tests the interaction between two plugins? I should be able to take it from there.

Reply:
> Can you give me an example of a test that tests the interaction between two plugins?

When we need to do that, we add the test to `spyder/app/tests/test_mainwindow.py`. So, I'd suggest to add here a simplified version of the `main_window` fixture we have in `spyder/app/tests/conftest.py`.

That shouldn't be that hard since you don't need to test as many things as in Spyder. But I think it's important because, as far as I can see, this plugin's connections to others are basically untested (i.e. all tests in this file are skipped now).